src/panels/lovelace/entity-rows/hui-media-player-entity-row.ts

```diff
@@ -68,47 +94,131 @@ class HuiMediaPlayerEntityRow extends LitElement implements LovelaceRow {
         .config=${this._config}
         .secondaryText=${this._computeMediaTitle(stateObj)}
       >
-        ${OFF_STATES.includes(stateObj.state)
+        ${Boolean(
+          OFF_STATES.includes(stateObj.state)
+            ? supportsFeature(stateObj, SUPPORT_TURN_ON)
+            : supportsFeature(stateObj, SUPPORT_TURN_OFF)
+        )
           ? html`
-              <div class="text-content">
-                ${this.hass!.localize(`state.media_player.${stateObj.state}`) ||
-                  this.hass!.localize(`state.default.${stateObj.state}`) ||
-                  stateObj.state}
-              </div>
+              <paper-icon-button
+                icon="hass:power"
```

Comment:
Should we show the turn off button when we're playing?

Reply:
Absolutely. I use it all the time with the min-media-player and is usually my only interaction with the row at all